tests/api/switch_api_test.py

```diff
@@ -320,7 +320,7 @@ def test_anonymous_switch_can_be_netman_proxied(self):
             username='root',
             password='password',
             port=None,
-            netman_server='1.2.3.4').once().ordered().and_return(self.switch_mock)
+            netman_server=['1.2.3.4']).once().ordered().and_return(self.switch_mock)
```

Comment:
It's good to point it out Mat, but like Martin, I think we are still at a point in the project where we don't need to worry about it.

This PR gets my :+1: